test/metabase/api/database_test.clj

```diff
@@ -372,7 +374,10 @@
     (testing "We should not include the saved questions virtual DB if there aren't any cards"
       (not-any?
        :is_saved_questions
-       ((mt/user->client :lucky) :get 200 "database?saved=true")))))
+       ((mt/user->client :lucky) :get 200 "database?saved=true")))
+    (testing "Ommit virtual DB if nested queries are disabled"
+      (tu/with-temporary-setting-values [enable-nested-queries false]
```

Comment:
Most of the test macros are in `mt/` now. So you could actually do `mt/with-temporary-setting-values` here instead of trying to remember what namespace it's in